initrd/bin/kexec-select-boot

```diff
@@ -1,8 +1,82 @@
 #!/bin/sh
 # Generic configurable boot script via kexec
-set -e -o pipefail
-. /tmp/config
-. /etc/functions
+
+usage() {
+	cat <<End-of-message
+Usage
+=====
+
+Generic configurable boot script via kexec
+	
+$0 -b /boot [options]
+	
+Options
+-------
+
+### -a
+
+Anything provided to the -a argument is appended to the kernel command line (usually comes from parsing the grub, syslinux, BLS files). Ex.
+
+	-a 'export CONFIG_BOOT_KERNEL_ADD="intel_iommu=on intel_iommu=igfx_off"'
+
+### -b
+
+path to boot files.  e.g.  -b /boot
+
+### -c = '*.cfg'
+
+specify the grub config file(s)
+
+### -d
+
+set the params device.  defaults to same as -b
+
+### -f
+
+force boot by considering hash and rollback checks as good
+
+### -g
+
+use graphical menus.  default is text only.
+
+### -h
+
+show help
+
+### -i
+
+ignore bad hash and/or bad rollback
```

Comment:
Those are implicit in code when required to boot insecurely, from boot options menu.
The script starts by setting valid_hashes and valid_rollback to n.

Otherwise, same thing here, the config options are coming from BOARD config, exported and evaluated here:
https://github.com/osresearch/heads/blob/0d5b3d75e7b84ca85ff252fe5fe23aea990da435/initrd/bin/kexec-select-boot#L289-L294

if the board config they are required but not present, booting will fail.

Reply:
How about
```
### -i

Use this parameter to ignore the hash and rollback security settings and boot a potentially compromised system.
```